src/views/Home.jsx

```diff
@@ -1,11 +1,22 @@
-import './Home.css';
+import { useState } from 'react';
 import { generateToken } from '@the-collab-lab/shopping-list-utils';
+import './Home.css';
+
+export function Home({ makeNewList, joinList, handleError, joinListErrorMsg }) {
+	const [inputValue, setInputValue] = useState('');
```

Comment:
I really like how @fakehouseplant and @drakenguyen4000 used state for handling changes made to an input on a Form in real-time (issue #4), so I adopted the same strategy for the joining an existing list Form.